src/Uno.UI.Tests/Windows_UI_XAML_Controls/SelectorTests/Given_Selector.cs

```diff
@@ -280,5 +280,16 @@ public void When_SelectionChanged_And_SelectorItem_IsSelected_Changed()
 			Assert.IsFalse(source[1].IsSelected);
 			Assert.IsFalse(source[2].IsSelected);
 		}
+
+
+		[TestMethod]
+		public void When_Selector_Created_Check_SelectedIndex()
+		{
+			var SUT = new Selector();
+			SUT.ItemsSource = new[] { Tuple.Create("1", 1), Tuple.Create("2", 2), Tuple.Create("3", 3) };
+			
+			Assert.IsFalse(SUT.SelectedIndex >= 3);
```

Comment:
I think the test should be setting an observable collection of 3 items, set a selected item to be the last, then remove the list item, then check that `SelectedIndex` is -1.

Also, by default, I don't think the Selector selects any item.

Reply:
Ok! Done!